### PR TITLE
Remove unecessary steps in cloudbuild.

### PR DESCRIPTION
**Pull request by kevensen**

### CODE REVIEW

File: cloudbuild/cloudbuild-pr.yaml [hunk 1]
```diff
@@ -30,15 +30,6 @@ steps:
   id: Setup connection to GKE
   args: ["container", "clusters", "get-credentials", "$_GKE_CLUSTER", "--project", "$_PROJECT_ID", "--region", "$_LOCATION"]
 
-- name: 'gcr.io/cloud-builders/docker'
```
Comment (hshin-g) [inline cloudbuild/cloudbuild-pr.yaml:33]: This will not have negative side effects?

Comment (kevensen) [reply cloudbuild/cloudbuild-pr.yaml:33]: No.  I was using these for debugging the Cloud Build process.

**Review by hshin-g — APPROVED** [review]

**Pull request merged**